cli/src/create.rs

```diff
@@ -319,7 +374,7 @@ fn copy_dir(input_dir: &Path, output_dir: &Path) -> anyhow::Result<()> {
     Ok(())
 }
 
-fn copy_keys(target_dir: std::path::PathBuf) -> Result<(), anyhow::Error> {
+fn copy_keys(target_dir: std::path::PathBuf) -> anyhow::Result<(), Error> {
```

Comment:
I think it can be
```suggestion
fn copy_keys(target_dir: std::path::PathBuf) -> anyhow::Result<()> {
```
?